floor/floor/controller/midi/functions.py

```diff
@@ -38,52 +39,77 @@ def add(cls, name, **kwargs):
 MidiFunctions.add('playlist_stop',
                   help_text='Stop playback of the current playlist.')
 MidiFunctions.add('playlist_goto_1',
+                  context_value=1,
                   help_text='Go to playlist position 1.')
 MidiFunctions.add('playlist_goto_2',
+                  context_value=2,
                   help_text='Go to playlist position 2.')
 MidiFunctions.add('playlist_goto_3',
+                  context_value=3,
                   help_text='Go to playlist position 3.')
 MidiFunctions.add('playlist_goto_4',
+                  context_value=4,
                   help_text='Go to playlist position 4.')
 MidiFunctions.add('playlist_goto_5',
+                  context_value=5,
                   help_text='Go to playlist position 5.')
 MidiFunctions.add('playlist_goto_6',
+                  context_value=6,
                   help_text='Go to playlist position 6.')
 MidiFunctions.add('playlist_goto_7',
+                  context_value=7,
                   help_text='Go to playlist position 7.')
 MidiFunctions.add('playlist_goto_8',
+                  context_value=8,
                   help_text='Go to playlist position 8.')
 MidiFunctions.add('playlist_goto_9',
+                  context_value=9,
                   help_text='Go to playlist position 9.')
 MidiFunctions.add('playlist_goto_10',
+                  context_value=10,
                   help_text='Go to playlist position 10.')
 MidiFunctions.add('playlist_goto_11',
+                  context_value=11,
                   help_text='Go to playlist position 11.')
 MidiFunctions.add('playlist_goto_12',
+                  context_value=12,
                   help_text='Go to playlist position 12.')
 MidiFunctions.add('playlist_goto_13',
+                  context_value=13,
                   help_text='Go to playlist position 13.')
 MidiFunctions.add('playlist_goto_14',
+                  context_value=14,
                   help_text='Go to playlist position 14.')
 MidiFunctions.add('playlist_goto_15',
+                  context_value=15,
                   help_text='Go to playlist position 15.')
 MidiFunctions.add('playlist_goto_16',
+                  context_value=16,
                   help_text='Go to playlist position 16.')
 MidiFunctions.add('set_bpm',
                   help_text='Set the global bpm based on note velocity or controller value.')
 MidiFunctions.add('set_brightness',
                   help_text='Adjust the max brightness of the floor')
 
+# Add a number of ranged values, likely faders or knobs, that can be used within processor code
+# to tweak parameters.
+for idx in range(1, 16):
```

Comment:
We could also start with fewer knobs; I was thinking a lot of real-world FX processors get away with 2 or maybe 4 knobs max - the scarcity of which might make processor designers more 'clever' about their use. Just a random idea, take it or leave it, doesn't cost us anything to keep it generous as is..